ophyd/tests/conftest.py

```diff
@@ -109,6 +108,7 @@ def fake_motor_ioc(prefix, request):
                step_size=f'{prefix}step_size',
                )
 
+    from caproto.tests.conftest import run_example_ioc
```

Comment:
we should use `pytest.importorskip` here so if you have tests that use this they fail gracefully rather than failing with ugly exceptions.